.pre-commit-config.yaml.jinja

```
@@ -47,6 +47,11 @@ repos:
         entry: found forbidden files; remove them
         language: fail
         files: "\\.rej$"
+  - repo: https://github.com/myint/autoflake
+    rev: v1.4
```

Please use a version variable (you have to define it also around line 5):

```suggestion
    rev: {{ repo_rev.autoflake }}
```

---

change done